data/campaigns/Eastern_Invasion/scenarios/06a_Undead_Crossing.cfg

```diff
@@ -0,0 +1,653 @@
+
+#define TURNS
+20 #enddef
+
+[scenario]
+    id=06a_Undead_Crossing
+    name= _ "Undead Crossing"
+    map_file=06a_Undead_Crossing.map
+    turns={TURNS}
+    next_scenario=07a_Capturing_the_Ogres
+    victory_when_enemies_defeated=no
+
+    {SCENARIO_MUSIC       battle.ogg}
+    {EXTRA_SCENARIO_MUSIC casualties_of_war.ogg}
+
+    {DEFAULT_SCHEDULE}
+
+    [story]
+        [part]
+            story=_ "In the west, where the Great River is deep and wide, the few crossings are known by all and heavily guarded. Hoping to evade detection, Gweddry and his band made for one of the numerous eastern fords instead..."
+            music,background=silence.ogg,story/S06a.jpg
+        [/part]
+    [/story]
+    {EI_TRACK {JOURNEY_06A_NEW} }
+
+    # wmllint: validate-off
+    [side]
+        side=1
+        controller=human
+        gold=150
+        team_name=good
+        user_team_name= _ "Wesnothians"
+        {FLAG_VARIANT loyalist}
+
+        # wmllint: recognize Gweddry
+        {CHARACTER_STATS_GWEDDRY}
+    [/side]
+    # wmllint: validate-on
+
+    {STARTING_VILLAGES 1 7}
+
+    [side]
+        side=2
+        type=Necromancer
+        id=Mal-un-Karad
+        name= _ "Mal-un-Karad"
+        canrecruit=yes
+        controller=ai
+        [modifications]
+            {TRAIT_STRONG}
+            {TRAIT_QUICK}
+        [/modifications]
+        recruit=Vampire Bat,Ghost,Wraith,Walking Corpse,Soulless
+        {GOLD 40 120 200}
+        {INCOME -3 1 5}
+        team_name=undead
+        user_team_name=_"Undead"
+        {FLAG_VARIANT undead}
+
+        {LOYAL_UNIT 2 ({ON_DIFFICULTY "Walking Corpse" "Skeleton"        "Revenant"})     30 5} {GUARDIAN}
+        {LOYAL_UNIT 2 ({ON_DIFFICULTY "Walking Corpse" "Skeleton"        "Revenant"})     22 10} {GUARDIAN}
+        {LOYAL_UNIT 2 ({ON_DIFFICULTY "Walking Corpse" "Skeleton Archer" "Bone Shooter"}) 31 10} {GUARDIAN}
+    [/side]
+    {RECRUIT_UNIT_VARIATIONS 2 "Walking Corpse" serpent,serpent,serpent,serpent}
+    {RECRUIT_UNIT_VARIATIONS 2 "Soulless" serpent,serpent,serpent,serpent}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 (Vampire Bat) 2}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 (Ghost) 3}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 (Wraith) 1}
+    {STARTING_VILLAGES 2 7}
+    {SILENTLY_LIMIT_LEADER_MOVES 2 3}
+
+    # when undead army has been whittled down, pull back to the central island
+    [event]
+        name=die
+        first_time_only=no
+        [filter]
+            side=2
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                side=2
+                count={ON_DIFFICULTY 0-6 0-8 0-10}
+            [/have_unit]
+        [/filter_condition]
+        [fire_event]
+            name=undead_retreat
+        [/fire_event]
+    [/event]
+    [event]
+        name=undead_retreat
+        [message]
+            speaker=Mal-un-Karad
+            message= _ "You know what, fine, you can have the south shore! I’m going to stay nice and safe in my castle."
```

Comment:
Could the first sentence be better punctuated as 'You know what? Fine! You can have the south shore!'

Reply:
Maybe even "You know what!?"

Reply:
Sure. Just noting the ordering of the punctuation seems to favour '?!' in this campaign. I don't mind either way as long as it's consistent and not necessarily duplicated (there are some instances of '?!?' and also '!?!', which I disagree with).